ckanext/ontario_theme/templates/internal/package/resource_validation.html

```diff
@@ -15,6 +15,13 @@
 {% endblock form_title %}
 
 {% block primary_content_inner %}
+  {% set package_id = resource.get('package_id') %}
+  {% set resource_id = resource.get('id') %}
+  {% set action = h.url_for('datastore.dictionary',
+      id=package_id,
+    resource_id=resource.id) %}
+  {% set is_csv = "True" %}
```

Comment:
`def resource_validation(id, resource_id):` in the plugin.py, passes the package id as `id` and you can pass the `resource_id` as well. Line 18 and 19 are redundant and you can use the id and resource_id for `set action`

Reply:
Actually `resource.id` is already in this page

Reply:
Good point. In fact, `resource_id` from line 19 is never actually used.